04-call-summaries/refinement.py

```diff
@@ -0,0 +1,65 @@
+import datetime
+
+from langchain_core.prompts import PromptTemplate
+
+from chunking import get_text_chunks
+from llm import select_client
+from run import get_transcript
+
+
+CHUNKING_PROMPT = """
+You are a helpful AI assistant tasked with summarizing transcripts, however we can only process the transcripts in pieces.
+Please fill out and return the following template: {template} with data in the text: {text}
+If the following template already has the field filled out, do not overwrite this information.
+"""
+
+initial_temp = """
```

Comment:
very minor nit: might be worth spelling out `template` to avoid any confusion with `temperature` or `temporary`